### PR TITLE
Wip/hughsie/dell dock regression2

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Comment by superm1** [comment]
This was my idea too in https://github.com/fwupd/fwupd/issues/1975#issuecomment-611214756.
Can you check and make sure that the plug / unplug without a firmware update works here?

**Comment by hughsie** [comment]
>  Can you check and make sure that the plug / unplug without a firmware update works here?

Yes, this fixes the unplug / replug.

File: data/device-tests/devices/dell-wd19tb-ec.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Dell WD19TB Dock (EC)",
+  "guids": [
+    "cd357cf1-40b2-5d87-b8df-bb2dd82774aa"
+  ],
+  "releases": [
+    {
+      "version": "01.00.00.04",
+      "file": "0f7990c89255596d6258c9715b59a40878d37c0ac3ef368aa01c99bed0c59a33-WD19FirmwareUpdateLinux_01.00.15.cab"
```
Comment (superm1) [inline data/device-tests/devices/dell-wd19tb-ec.json:9]: Hmm this is actually leaking the full filename; IE someone could easily just prepend https://fwupd.org/downloads/.  Could you just make a local symlink for now to obfuscate it?

File: data/device-tests/devices/dell-wd19tb-main.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Dell WD19TB Dock (Main)",
```
[the remark this answers is not in the document]
Comment (hughsie) [reply data/device-tests/devices/dell-wd19tb-main.json:2]: so `s/main/pkgver` perhaps?

File: data/device-tests/devices/dell-wd19tb-ec.json (hunk 1 above)
Comment (hughsie) [reply data/device-tests/devices/dell-wd19tb-ec.json:9]: Fixed. If you run that test you'll have to download it manually and rename it too.

File: data/device-tests/devices/dell-wd19tb-main.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (superm1) [reply data/device-tests/devices/dell-wd19tb-main.json:2]: Yeah

File: data/device-tests/devices/dell-wd19tb-ec.json (hunk 1 above)
Comment (superm1) [reply data/device-tests/devices/dell-wd19tb-ec.json]: OK thanks.  When it goes stable we can revert it back to this too.

**Comment by superm1** [comment]
Just tested on my side, looks good now.

**Pull request merged**